### DISCUSSION TITLE
Any examples for using the processors?

### DISCUSSION THREAD

**Discussion by jiakuan** [treblereel/gwt3-processors#23] [Q&A] [answered]
I tried it in a Maven project following some configuration in the tests, but the generated JS was empty. I haven't had a chance to investigate more, just wondering if there are any workable examples with typical configuration? That'd be really helpful.

**Comment by treblereel** [comment]
@jiakuan could you give me a link to your demo project?

**Reply by treblereel** [reply]
@jiakuan you can take a look at the tests, each of them is a workable example

**Reply by jiakuan** [reply]
I've created a sample project, and documented the steps I did in the README:
https://github.com/jiakuan/test-j2cl

**Reply by treblereel** [reply]
thank you, I ll take a look today

**Reply by treblereel** [reply, marked as answer]
@jiakuan, you use 0.21-SNAPSHOT, but the correponding gwt3-processors (0.6) has been released only yesterday. Here is a fixed demo
[test-j2cl.zip](https://github.com/treblereel/gwt3-processors/files/12489402/test-j2cl.zip)

**Reply by jiakuan** [reply]
Thanks a lot. That worked.
